_posts/2018-02-27-release-1.4.2.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Apache Flink 1.4.2 Released"
+date:   2018-02-27 9:00:00
```

Comment:
Please change to `date:   2018-02-27 12:00:00` to keep the dates of the blog posts consistent. Thanks!